lts.json

```diff
@@ -14,5 +14,5 @@
       "ember-d3": "0.2.0"
```

Comment:
Can you please check the full content of this file? I don't know what happened with my previous PR. I got merging issues.